src/main/java/edu/harvard/iq/dataverse/export/ExportService.java

```diff
@@ -312,14 +328,18 @@ private InputStream getCachedExportFormat(Dataset dataset, String formatName) th
         InputStream cachedExportInputStream = null;
 
         try {
+		/*
             if (dataAccess.getAuxFileAsInputStream("export_" + formatName + ".cached") != null) {
                 cachedExportInputStream = dataAccess.getAuxFileAsInputStream("export_" + formatName + ".cached");
                 return cachedExportInputStream;
             }
+	    */
+		cachedExportInputStream = dataAccess.getAuxFileAsInputStream("export_" + formatName + ".cached");
+		return cachedExportInputStream; //unclear why this was checking for null before return, if the failure condition is returning null
```

Comment:
Better remove the commented out code. My guess is that the programmes wat thinking of doing something more with the result, but didn't. The function documentation should tell that the result can be null, that should be sufficient information, allthough no javadoc is used.

Reply:
The only reason I could think of to leave the commented out code would be "don't try to do it like this next time" - but that would need a clarifying comment in addition, and probably isn't worth the lack of clarity.

It's unfortunate that the javadoc habit hasn't been more widely adopted.

@PaulBoon Thanks for the feedback!